source/lovelace/views.markdown

```diff
@@ -62,6 +62,8 @@ views:
 
 You can link to one view from another view by its ID. For this use cards that support navigation (`navigation_path`). Do not use special characters in IDs.
 
+IDs are also used to identify a view for editing, we will add an ID to your view if you don't specify it. You can change this ID, as long as every view has an unique ID.
```

Comment:
I think it will help if you clarify the purpose of these IDs. May be we can just indicate that the IDs will be used for things like rearranging UI using drag-and-drop, etc.